active_directory/tests/test_active_directory.py

```diff
@@ -10,14 +10,11 @@
 from datadog_checks.active_directory import ActiveDirectoryCheck
 from datadog_checks.active_directory.active_directory import DEFAULT_COUNTERS
 
-# for reasons unknown, flake8 says that pdh_mocks_fixture is unused, even though
```

Comment:
Do we want to still leave these comments since we're still ignoring F401?

Reply:
good question

Reply:
No, since we know why https://github.com/DataDog/integrations-core/pull/3060#discussion_r252761334